server/settings/publishers.py

```diff
@@ -1088,7 +1097,9 @@ class PublishersModel(BaseSettingsModel):
         "sync_workfile_version": False
     },
     "CollectFbxAnimation": {
-        "enabled": False
+        "enabled": True,
+        "optional": True,
+        "active": False
```

Comment:
@moonyuet How about preserving the old behavior and having to need to enable this explicitly in settings? easier for backwards compatibility.
```suggestion
        "enabled": False,
        "optional": True,
        "active": True
```